app/views/admin/members/_search_results.erb

```diff
@@ -0,0 +1,53 @@
+<div class="columns member-lookup-items">
+  <div class="column col-12">
+    <h2 id="checkout" class="h4">Lend or Hold Item</h2>
+      <%= portal do %>
+        <% @items_by_name.each do |item| %>
+          <div class="columns">
+            <div class="column col-8">
+              <h5>
+                <%= full_item_number(item) %>
+                <strong><%= item.name %></strong>
+              </h5>
+
+              <% if item.checkout_notice.present? %>
+                <div class="info-box">
+                  <p><%= item.checkout_notice %></p>
+                </div>
+              <% end %>
+
+              <% if item.holds.active.any? %>
+                <div class="toast">
+                  <p>
+                    This item has been
+                    <%= link_to "placed on hold by #{pluralize(item.holds.active.count, "person")}",
+                          admin_item_item_holds_path(item) %>.
+                  </p>
+                </div>
+              <% end %>
+
+              <p>
+                <%= render partial: "admin/loans/search_form", locals: {item: item} %>
+              </p>
+
+              <p>
+                <%= render partial: "admin/members/holds/search_form", locals: {item: item} %>
+              </p>
+            </div>
+
+            <div class="column col-4">
+              <% if item.image.attached? %>
+                <%= link_to admin_item_path(item) do %>
+                  <%= image_tag url_for(rotated_variant(item.image, resize_to_limit: [200, 140])), class: "p-centered" %>
+                <% end %>
+              <% else %>
+                <div class="image-placeholder"></div>
+              <% end %>
+
+              <%= link_to "Cancel", admin_member_path(@member, anchor: "checkout"), class: "btn" %>
```

Comment:
I think we'll want to pull this out of the loop so there is only one cancel button per set of search results.